lib/ansible/plugins/inventory/linode.py

```diff
@@ -29,6 +29,11 @@
             required: true
             env:
                 - name: LINODE_ACCESS_TOKEN
+        instance_access:
```

Comment:
Instead of hardcoding this choice, why not use constructed/compose functionality to allow user to set `ansible_host` and similar variables to anything they need.

Reply:
Can you explain a bit more how this would work? I've dug around a bit and am not sure.

Reply:
basically #53869 does same for vultr plugin

Reply:
Does make sense to use `ansible_host` rather than change the host identifier. I'll make that change.

Reply:
yes, since that is what you specifically are doing with the new option 'setting ansible_host' while the compose system gives you a generic way of setting any/all kinds of variables, you really only need this type of option for 'inventory_hostname'

Reply:
Thanks for following up, this does use `ansible_host` now.

Reply:
maybe I am not being clear, my point is you do NOT need THIS specific option, using compose any user can decide exactly what `ansible_host` value is set to, as well as any other connection or non connection specific variable.